docs/documentation/server_admin/topics/organizations/managing-organization.adoc

```diff
@@ -0,0 +1,38 @@
+[id="managing-organization_{context}"]
+
+=  Managing an Organization
+[role="_abstract"]
+Organizations can be managed once you enable the `Organizations` setting in the `Realm Settings` page.
+
+Once you enable this setting, the `Organizations` section is available from the left side menu.
+
+By clicking on the `Organizations` section you can start creating an organization by clicking on the `Create Organization`
+button. If organizations already exist, you will see a list with all of them and options to search, edit, or delete an organization.
```

Comment:
```suggestion

You enable the Organizations feature for the current realm.

.Procedure

. Click `Realm Settings` in the menu.

. Click *Create Organization*.
+
If organizations already exist, you see a list of organizations and options to search, edit, or delete an organization.
```

Reply:
Pedro, for accessibility reasons, we avoid using  directional words such as left or right, because it makes no sense to a visually impaired user.

Reply:
Makes sense, will keep that in mind.